Breeze.UI/src/interfaces/itransaction.ts

```diff
@@ -0,0 +1,10 @@
+export interface ITransaction {
```

Comment:
https://angular.io/guide/styleguide

Do name an interface using upper camel case.

Consider naming an interface without an I prefix.

Consider using a class instead of an interface for services and declarables (components, directives, and pipes).

Consider using an interface for data models.

`Why? TypeScript guidelines discourage the I prefix.`

Why? A class alone is less code than a class-plus-interface.

Why? A class can act as an interface (use implements instead of extends).

Why? An interface-class can be a provider lookup token in Angular dependency injection.